src/en/ReadLightNovel.lua

```diff
@@ -79,13 +79,15 @@ return {
 			title = doc:selectFirst(".block-title h1"):text(),
 			imageURL = left:selectFirst(".novel-cover img"):attr("src"),
 			description = table.concat(map(details:selectFirst(".novel-detail-body"):select("p"), text), "\n"),
-			alternativeTitles = map(details:selectFirst(".novel-detail-item.color-gray"):select("li a"), text),
 			status = ({
 				Ongoing = NovelStatus("PUBLISHING"),
 				Completed = NovelStatus("COMPLETED")
 			})[leftdetails:get(leftdetails:size()-1):selectFirst("li"):text()],
 			language = leftdetails:get(3):selectFirst("li"):text()
 		}
+		if details:selectFirst(".novel-detail-item.color-gray") ~= nil then
+			info.setAlternativeTitles(info, map(details:selectFirst(".novel-detail-item.color-gray"):select("li a"), text))
```

Comment:
`info.setAlternativeTitles` should be `info:setAlternativeTitles`

Reply:
I think that should have been fine, but I changed it to use the colon parameter instead of manually self referencing.